src/modapi/moditem.h

```diff
@@ -72,4 +73,56 @@ struct CModAPI_ModItem_LineStyle
 	int m_AnimationSpeed;
 };
 
+enum ESkinModifier
```

Comment:
I think it's useless, no ? We can simply add a function that fill each part according to one of the predefined json file.